src/core/archive.ts

```diff
@@ -549,6 +549,8 @@ class ArchiveDownloader extends Downloader {
         this.allDownloadTasks = previousTask.allDownloadTasks;
         this.downloadTasks = previousTask.downloadTasks;
         this.finishedFilenames = previousTask.finishedFilenames;
+
+        logger.setDebugMode(this.verbose);
```

Comment:
I think this should be moved into `index.ts` to align with the download mode.

Reply:
BTW. I think `enableDebugMode` is better than `setDebugMode` because the design allows turning on debug mode but not off.

Reply:
Q1: Yeah actually loading options through `resume()` is not a good idea, as it misses the opportunity to properly set everything up when initializing the `ArchiveDownloader`. This means that the **previous-task parsing procedure** should also be moved to the `Erii.bind()`. I’m literally new to TypeScript and any serious JavaScript programming. Would you like to fix it yourself?

Q2: It may be a bit further along. Regarding the improvement of the resume command, have you considered any command-line override issue? (e.g., `verbose: true` written in `tasks.json` that conflicts with `minyami -r ...`, as the user wants to turn off the verbose feature for the resume run, or vice versa.)

Reply:
I've reconsidered this issue. I believe `verbose` should be a one-time parameter and not saved in the task info; only the `--verbose` command will take effect. I'll merge this PR first, then make some modifications myself.